src/test/java/picard/sam/SamFileConverterTest.java

```diff
@@ -32,7 +33,6 @@
 import java.util.List;
 
 import static org.testng.Assert.assertEquals;
-import static org.testng.Assert.assertTrue;
 
 public class SamFileConverterTest {
```

Comment:
It looks like this should be named `SamFormatConverterTest`.

Reply:
Done in previous round.